### DISCUSSION TITLE
Documentation: Content guidelines - tutoyeren

### DISCUSSION THREAD

**Discussion by JeroenduC** [nl-design-system/backlog#207] [Richtlijnen]
Bij tekst en taalgebruik is het interessant om een uitspraak te doen over het gebruik van tutoyeren in content. Het lijkt mij handig daar naar te kunnen verwijzen en zo discussies snel te kunnen beslechten.

Volgens mij gebruiken de meeste gemeentes 'u'. Alhoewel het wat formeler en ouderwetser overkomt, is het voor mensen uit alle culturen goed te begrijpen en zal het ook voor ouderen netjes overkomen.

**Comment by JeroenduC** [comment]
Er staat al het een en ander over de aanspreekvorm, maar er wordt nog geen echte keuze gemaakt. Wellicht omschrijven onder welke omstandigheden welke keuze relevant is.

**Comment by usethetics** [comment]
Interessant! Ik denk echter wel dat neerkomt op de tone of voice en merkidentiteit van specifieke organisaties en/of producten. Een CJIB bijvoorbeeld zal wellicht mensen op een andere manier aan willen spreken dan hoe een gemeenteproduct gericht op jongeren dat doet.

Ik denk dan ook dat hier eenzelfde soort aanpak als de visuele stijl genomen moet worden; kernprincipes zitten in het NL Design System, voor organisatie-specifieke principes blijven organisaties zelf verantwoordelijk.

Het lijkt me wel goed om ook even een aantal content specialisten hier over te laten buigen, wellicht zijn er duidelijke richtlijnen te documenteren wanneer ‘u’ vs. ‘jij’ gebruikt dient te worden.

**Comment by eslook** [comment]
Naar aanleiding van een [thread in Slack](https://codefornl.slack.com/archives/C7KDQRXMZ/p1761562567504419), zie ook:
- [Als je gaat schrijven - Toolkit Taal](https://toolkittaal.gebruikercentraal.nl/richtlijnen/als-je-gaat-schrijven/#taal-stijl-en-grammatica)
- [Schrijf persoonlijk: ik, we, u of jij | Schrijfwijzer](https://www.amsterdam.nl/schrijfwijzer/heldere-taal-basis-onze-huisstijl/taalgebruik-heldere-taal/schrijf-persoonlijk-we-jij/)
